lib/vdsm/storage/managedvolume.py

```diff
@@ -112,18 +113,18 @@ def attach_volume(vol_id, connection_info):
                         "Unsupported volume type, supported types are: "
                         "rbd, iscsi")
```

Comment:
need to support `lightos` here, assuming it's the short name we care about - the full name is `cinder.volume.drivers.lightos.LightOSVolumeDriver`. Otherwise I get this from the engine:

```
2022-04-19 07:53:03,899-04 WARN  [org.ovirt.engine.core.vdsbroker.vdsbroker.AttachManagedBlockStorageVolumeVDSCommand] (EE-ManagedThreadFactory-engine-Thread-85595) [4ddaf7c2] Unexpected return value: Status [code=2009, message=Unsupported operation: 'reason=Unsupported volume type, supported types are: rbd, iscsi, context={}']
2022-04-19 07:53:03,899-04 ERROR [org.ovirt.engine.core.vdsbroker.vdsbroker.AttachManagedBlockStorageVolumeVDSCommand] (EE-ManagedThreadFactory-engine-Thread-85595) [4ddaf7c2] Failed in 'AttachManagedBlockStorageVolumeVDS' method
```

Reply:
yes, this is the `driver_volume_type` property, I think the full name would be `volume_driver` or something, but it doesn't matter in this context.

Adding `lightos` to the tuple is enough to make this work? If it is, I believe we can move forward with this PR

Reply:
I wish... adding `lightos` is necessary to make progress, but we still hit a failure in the engine. I think I see maybe why it happens, will commment on the engine patch.

Reply:
Well, the current engine code expects the "path" property, so I suppose this is why it fails

My PR went stale, but I rebased and started a build https://github.com/oVirt/ovirt-engine/actions/runs/2189742137
RPMs should be available in about 20-30 minutes (assuming it doesn't fail :)

Reply:
That's great, thanks. I'll give it a spin when it will be ready and let you know how it goes.